src/storage/storage.bif

```diff
@@ -0,0 +1,18 @@
+##! Functions related to general storage operations. These are not specific to async or sync.
+
+%%{
+
+#include "zeek/storage/Backend.h"
+
+using namespace zeek;
+using namespace zeek::storage;
+
+%%}
+
+module Storage;
+
+function Storage::is_connected%(backend: opaque of Storage::BackendHandle%) : bool
```

Comment:
I know you only needed this for testing - but BIFs really should have documentation attached to them, even if we only use them for our tests.